core/api/v1/keytransparency.proto

```diff
@@ -187,6 +195,35 @@ message ListEntryHistoryResponse {
   int64 next_start = 2;
 }
 
+// BatchListUserRevisionsRequest contains a list of users and a range of revisions.
+message BatchListUserRevisionsRequest {
+  // directory_id identifies the directory in which the user lives.
+  string directory_id = 1;
+  // user_ids are the user identifiers.
+  repeated string user_ids = 2;
```

Comment:
what's the behavior if a user_id doesn't exist in the map? Does the API handle that gracefully using an empty leaf or is it an error case we should call out?

Reply:
Good question.  Queries for leaves that don't exist in the Trillian Map produce proofs to empty map leaves, not gRPC errors.